src/components/Offers/Edit/EditOfferForm.js

```diff
@@ -1,14 +1,21 @@
 import React, { useCallback, useContext, useEffect } from "react";
-import { parseRequestErrors } from "./New/CreateOfferUtils";
-import OfferForm from "./New/form-components/OfferForm";
-import { editOffer } from "../../services/offerService";
+import { parseRequestErrors } from "../New/CreateOfferUtils";
+import OfferForm from "../New/form-components/OfferForm";
+import { editOffer } from "../../../services/offerService";
 import { Redirect, useParams } from "react-router-dom";
-import useOffer from "../../hooks/useOffer";
-import useOfferForm from "../../hooks/useOfferForm";
-import { INITIAL_JOB_DURATION } from "../../reducers/searchOffersReducer";
+import useOffer from "../../../hooks/useOffer";
+import useOfferForm from "../../../hooks/useOfferForm";
+import { INITIAL_JOB_DURATION } from "../../../reducers/searchOffersReducer";
+import useSession from "../../../hooks/useSession";
 
 export const EditOfferControllerContext = React.createContext();
 
```

Comment:
Don't forget that about offer states that stop the offer from being edited (some time has passed after the publish date (1h?) or the offer was disabled by admin, for example (there might be more, I'm just recalling these 2)

Reply:
[Time passed publish date verification](https://github.com/NIAEFEUP/nijobs-fe/blob/3e18cf0b864170103cffaf2fd08c4a40d5cfb9e8/src/components/Offers/Form/OfferUtils.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L50-L56)
I'll add verification of disabled